### ISSUE TITLE
[Help Wanted]Problem while training on customize datasaet

### ISSUE THREAD

**Issue by laplaceson** [zylo117/Yet-Another-EfficientDet-Pytorch#632]
Hi @zylo117 ,
First thanks  for your awesome work!
I try to use pretrained  weight  D0 model on my custom dataset.
However, it always gives reg loss 0 . So i try to use other method to modify the anchors_scales and anchors_ratios.
(Since my object are small and sparse). The reg loss goes non-zero and converge.
But i figure out my validation reg loss and training reg loss have big difference.
![image](https://user-images.githubusercontent.com/20430525/115993766-da744680-a606-11eb-8091-7fb48e33437b.png)
And finally i use debug mode (Or using weight to efficientdet_test.py ) to visualize it.
But get nothing detected. Do you have any suggestion? Thanks 
By the way, my task is single class object detection(object size often not big).

**Comment by laplaceson** [comment]
Another question, i see some of your comment says it's hard to detect smaller 32 * 32 , is this means that 
under 512 * 512 's image, if the annotation is smaller than 32 * 32, it's hard to detect in this network? Thanks

**Comment by zylo117** [comment]
32x32 under 512x512
